### ISSUE TITLE
Does D1-D3 supported for now?

### ISSUE THREAD

**Issue by jinfagang** [signatrix/efficientdet#14]
I have trained another implementation of D1 and D3 but all divergent, will this implementation as well?

**Comment by uvipen** [comment]
We are training D1 :)

**Comment by jinfagang** [comment]
Hoping mAP updates

**Comment by quangtn266** [comment]
> Hoping mAP updates
@jinfagang, do you get results for d1 model or its pre-trained model? if you have it, can you share it with me?

**Comment by vakkov** [comment]
Hi! Do you have any progress or tips on the hyperparameter values when training D1?

**Comment by CraigWang1** [comment]
@vakkov I tried training D1 on a custom dataset by

1) changing the backbone from efficientnet-b0 to efficientnet-b1 in EfficientNet class
2) changing compound_coef in EfficientDet class from 0 to 1
3) changing common_size to equal 640 in resizer class

However, although the mAP was higher than D0 on my custom dataset, it didn't generalize to new images at all, which is a major concern. Also, the training reached its lowest loss values about 10 epochs in which was also not expected.

This is what I did; have you had any similar experiences?

**Comment by CraigWang1** [comment]
@uvipen Has there been any progress training D1/new developments or news about when to expect support for D1?

Thank you very much.

**Comment by AlhasanAlshaebi** [comment]
> @vakkov I tried training D1 on a custom dataset by
> 
> 1. changing the backbone from efficientnet-b0 to efficientnet-b1 in EfficientNet class
> 2. changing compound_coef in EfficientDet class from 0 to 1
> 3. changing common_size to equal 640 in resizer class
> 
> However, although the mAP was higher than D0 on my custom dataset, it didn't generalize to new images at all, which is a major concern. Also, the training reached its lowest loss values about 10 epochs in which was also not expected.
> 
> This is what I did; have you had any similar experiences?

Hi @CraigWang1 How can I know the input size of images for any D0 to D7, as you said, you changed common_size to equal 640 in resizer class for D1. Based on what you Resize them. Any resource for that?
